client/extensions/woocommerce/woocommerce-services/views/packages/add-package.js

```diff
@@ -0,0 +1,245 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { concat, difference, flatten, map, omit, trim } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import FormSectionHeading from 'components/forms/form-section-heading';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
+import Dialog from 'components/dialog';
+import AddPackagePresets from './add-package-presets';
+import checkInputs from './modal-errors';
+import inputFilters from './input-filters';
+import FieldError from '../../components/field-error';
+
+const getDialogButtons = ( mode, dismissModal, savePackage, onRemove, translate ) => {
+	const buttons = [
+		<FormButton onClick={ savePackage }>
+			{ ( 'add' === mode ) ? translate( 'Add package' ) : translate( 'Apply changes' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **56** times:
`translate( 'Apply Changes' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/16409-packages-settings)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).